apps/api-documenter/src/cli/BaseAction.ts

```diff
@@ -72,28 +72,33 @@ export abstract class BaseAction extends CommandLineAction {
   // to apply all @inheritDoc tags before the .api.json file is written.
   // See DocCommentEnhancer._applyInheritDoc() for more info.
   private _applyInheritDoc(apiItem: ApiItem, apiModel: ApiModel): void {
-
     if (apiItem instanceof ApiDocumentedItem) {
       if (apiItem.tsdocComment) {
         const inheritDocTag: tsdoc.DocInheritDocTag | undefined = apiItem.tsdocComment.inheritDocTag;
 
         if (inheritDocTag && inheritDocTag.declarationReference) {
           // Attempt to resolve the declaration reference
-          const result: IResolveDeclarationReferenceResult
-            = apiModel.resolveDeclarationReference(inheritDocTag.declarationReference, apiItem);
+          const result: IResolveDeclarationReferenceResult = apiModel.resolveDeclarationReference(
+            inheritDocTag.declarationReference,
+            apiItem
+          );
```

Comment:
This is the parameter style I like. I wish Prettier had an option to always do this if any of the parameters ever wrap.